backend/src/cache/deployer/deploy-cache-service.sh

```diff
@@ -22,7 +22,7 @@ set -ex
 
 echo "Start deploying cache service to existing cluster:"
 
-NAMESPACE=${NAMESPACE_TO_WATCH:-default}
+NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
```

Comment:
it's not necessary as external deployment passed in the value.
it's more for better default value

Reply:
Does any deployment method use `default` namespace?